pkg/hostagent/mount.go

```diff
@@ -48,7 +48,7 @@ func (a *HostAgent) setupMount(ctx context.Context, m limayaml.Mount) (*mount, e
 		RemotePath: expanded,
 		Readonly:   !m.Writable,
 		// NOTE: allow_root requires "user_allow_other" in /etc/fuse.conf
```

Comment:
This comment line needs to be updated too

Reply:
Sure